packages/next-app/src/contexts/InternalManagerContext.tsx

```diff
@@ -44,12 +61,31 @@ export function InternalManagerProvider({ children }: PropsWithChildren) {
   const [transactionStatus, setTransactionStatus] = useState<TransactionStatus>(
     TransactionStatus.AUTHORIZING
   );
+  const [tokenList, setTokenList] = useState<tokenListItem[]>([]);
+
+  const { networkConnectedToWallet } = useNetworks();
 
   function clearNotification() {
     setNotification(null);
     setIsNotifierOpen(false);
   }
 
+  useEffect(() => {
+    async function getTokensData() {
+      const res = await fetch(
+        "https://raw.githubusercontent.com/balancer/tokenlists/main/generated/balancer.tokenlist.json"
```

Comment:
could you try this token list? 

https://raw.githubusercontent.com/balancer/tokenlists/main/generated/listed-old.tokenlist.json

Reply:
https://linear.app/bleu-llc/issue/BAL-373/find-alternative-for-promiseall